kolibri/plugins/style_guide/assets/src/views/content/components/textbox.vue

```diff
@@ -4,13 +4,16 @@
     Textbox
 
```

Comment:
I've renamed this file as well. 

https://github.com/learningequality/kolibri/pull/1627/